app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>SearchApp</title>
+  <title>NEOS Catalogue</title>
```

Comment:
Noticed the title of our application layout....renamed to something that makes sense. Probably a good indication that this application layout is never used.

Reply:
:+1: